doc/content/devices/the-things-uno/_index.md

```diff
@@ -0,0 +1,432 @@
+
+### From the LoRaWAN Device Repository 
+This is the easiest way to register your Things Uno with the Things Stack.
+
+On the **Register end device** page, in the **From the LoRaWAN Device Repository** tab, under **Select the end device**, select the following mandatory fields:
+- Brand - `The Things Products`
+- Model – `The Things Uno`
+- Hardware Ver. – `1.0`
+- Firmware Ver. – `quickstart` or `abp` (we recommend you to use `quickstart` as it provides **OTAA**.)
+- Profile (Region) – Choose `EU_863_870` or `US_902_928` to match with your board. 
+
+{{< figure src="RegisterEndDeviceRepo.png" alt="Register End Device from LoRaWAN repository" >}}
+
+Under the **Enter registration data**, select/fill the following mandatory fields.
+
+If you have selected `quickstart` from the **Firmware Ver.** in the previous step:
+- Frequency plan – Select `Europe 863-870 MHz (SF9 for RX2 - recommended)` for **EU_863_870** or `United States 902-928 MHz, FSB2 (used by TTN)` for **US_902_928**.
+- AppEUI – Copy the **AppEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- DevEUI – Copy the **DevEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- AppKey – Select **Generate** button to generate an **AppKey**.
+- End device ID – A unique human-readable identifier for your Things Uno.
+
+{{< note >}}
+Following are the rules for creating an End device ID:
+- Must be unique within an application server of the network.
+- Must start with a lowercase letter or a number.
+- Can only contain lowercase letters, numbers, and hyphens.
+- Cannot have spaces between words. You can combine them using hyphens.
+- Must have less than 36 characters.
+{{</ note >}}
+
+Select the **Register end device** button. 
+
+{{< figure src="RegisterEndDeviceRepoOTAA.png" alt="Register End Device from LoRaWAN repository OTAA" >}}
+
+If you have selected `abp` from the **Firmware Ver.** in the previous step:
+- Frequency plan – Select `Europe 863-870 MHz (SF9 for RX2 - recommended)` for **EU_863_870** or `United States 902-928 MHz, FSB2 (used by TTN)` for **US_902_928**.
+- Device address - Select **Generate** button.
+- AppSKey– Select **Generate** button.
+- NwkSKey– Select **Generate** button.
+- End device ID – A unique human-readable identifier for your Things Uno (see above for naming conversions).
+
+Select the **Register end device** button. 
+
+{{< figure src="RegisterEndDeviceRepoABP.png" alt="Register End Device from LoRaWAN repository ABP" >}}
+
+Once registered, you will be redirected to the overview page of the newly registered device, where you can find the generated **AppKey** which we’ll need next.
+
+### Manually
+Manual registration requires providing more information than the previous option but it provides a flexible way of adding end devices. 
+
+On the **Register end device** page select the **Manually** tab.
+
+In the **Basic settings** select/fill the following:
+
+- LoRaWAN version - `MAC V1.1`
```

Comment:
```suggestion
- LoRaWAN version - `MAC V1.1`
```

This is incorrect, Uno uses MAC V1.0.2 and PHY 1.0.2 Rev B

Reply:
corrected